test/over_react/util/class_names_test.dart

```diff
@@ -239,6 +239,65 @@ main() {
             }));
           });
         });
+
+        group('merge', () {
+          test('returns className merged from second builder instance', (){
+            ClassNameBuilder otherBuilder = new ClassNameBuilder();
```

Comment:
#nit for local variables like this, the convention is to omit the typing on the left-hand side (LHS)

See more info here: https://www.dartlang.org/guides/language/effective-dart/design#avoid-type-annotating-initialized-local-variables